frontend/packages/knative-plugin/src/utils/create-eventsources-utils.ts

```diff
@@ -130,6 +153,16 @@ export const getEventSourceData = (source: string) => {
       },
       serviceAccountName: '',
     },
+    containersource: {
+      containers: [
+        {
+          image: '',
+          name: '',
+          args: [{ name: '' }],
+          env: [],
+        },
+      ],
```

Comment:
can we have 

```
template: {
    spec: {
      containers: [
        {
          image: '',
          name: '',
          args: [{ name: '' }],
          env: [],
        },
      ]
  }
}
```

then we can pass it as is to `getEventSourcesDepResource` and avoid  util `getContainerSourceResource`

Reply:
We cannot avoid the util function as MulticolumnField value return an array objects
ie: `args: [ {name: 'x'} ]` which is needs transformation in that util function to make it array of strings. eg: `args: ['x'].